src/services/db/GitHubService.ts

```diff
@@ -209,18 +207,14 @@ export default class GitHubService {
 
   async read(
     sessionData: UserWithSiteSessionData,
-    {
-      fileName,
-      directoryName,
-      branchName = STAGING_BRANCH,
-    }: { fileName: any; directoryName: any; branchName?: string }
+    { fileName, directoryName }: { fileName: any; directoryName: any }
```

Comment:
nit: could we stricten to `string`? i'm ok w leaving as-is and having another PR tidying types (similarly for type related nits gg forward in this PR)

Reply:
I change those that can easily change ah (ie change + ts dont complain) will deprior those that require a change in control flow for now